indico/modules/rb_new/client/js/components/TimeRangePicker.jsx

```diff
@@ -78,42 +114,18 @@ export default class TimeRangePicker extends React.Component {
         };
     }
 
+    static getDerivedStateFromProps({allowPastTimes, startTime}, prevState) {
```

Comment:
ugh can't we avoid using derived state here? :/

unless the options are expensive to generate i guess we could just generate them when needed during rendering?

Reply:
It was because of the fact that options are stored in the state. But now, when I looked at it closely I might be able to get rid of that.